Makefile

```diff
@@ -18,8 +18,15 @@ build:
 	$(eval PACKAGE := github.com/runfinch/finch-daemon)
 	$(eval VERSION ?= $(shell git describe --match 'v[0-9]*' --dirty='.modified' --always --tags))
 	$(eval GITCOMMIT := $(shell git rev-parse HEAD)$(shell if ! git diff --no-ext-diff --quiet --exit-code; then echo .m; fi))
+ifneq ($(STATIC),)
```

Comment:
This build recipe is becoming unnecessarily complex now and need some refactor.
Suggest moving the variables outside this recipe so that we can get rid of the `eval`. And we can have a more concise definition of the build flags.